test/unit/model/oasis/schema/Function.js

```diff
@@ -47,24 +47,28 @@ const schema = {
 };
 
 describe("Function", function () {
+  let functionImportType;
+  beforeEach(() => {
+    functionImportType = new FunctionDef(sampleMD);
+  });
   describe("#constructor()", function () {
     it("initializes properties", function () {
-      let fn = new FunctionDef(sampleMD);
-      assert.equal(fn.raw, sampleMD);
-      assert.equal(fn.name, "Function1");
-      assert.ok(fn.isBound);
-      assert.ok(fn.isComposable);
-      assert.equal(fn.entitySetPath, "path");
-      assert.ok(_.isArray(fn.parameters));
+      functionImportType = new FunctionDef(sampleMD);
```

Comment:
I would prefer to stick with naming from specification, or functionType to avoid naming collision in JS. But it's not important in unit tests.